vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteImageBlockController.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import edu.asu.diging.vspace.core.exception.BlockDoesNotExistException;
+import edu.asu.diging.vspace.core.services.IContentBlockManager;
+
+@Controller
+public class DeleteImageBlockController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Autowired
+    private IContentBlockManager contentBlockManager;
+
+    @RequestMapping(value = "/staff/module/{moduleId}/slide/{id}/image/{blockId}", method = RequestMethod.DELETE)
+    public ResponseEntity<String> deleteImageBlock(@PathVariable("blockId") String blockId) throws IOException {
+        try {
+            contentBlockManager.deleteImageBlockById(blockId);
+        } catch (BlockDoesNotExistException e) {
+            logger.error("Could not delete text block.", e);
```

Comment:
why does the error message say "couldn't delete text block" when it's deleting images.

Reply:
I would also not log it as an error, since it's more informational and make the error message say something that sounds less "error-like". It's not critical if someone tries to delete something that doesn't exist.